### PR TITLE
Fleet: Add new mappings for .fleet-actions signing

### PR DESCRIPTION
**Pull request by aleksmaus**
## What does this PR do?

Adds new ```signed``` mapping with ```data``` and ```signature``` in order to support the .fleet-actions documents signing.
This is a part of the the Agent Tamper Protection effort by the "Security Defends Workflows" team.

## Screenshots

Verified the ```signed``` mappings:
Screenshot 2023-02-14 at 3 30 27 PM

Verified the document is stored with the new properties correctly:
Screenshot 2023-02-14 at 3 29 55 PM

### CODE REVIEW

**Review by pzl — APPROVED** [review]

**Comment by ywangd** [comment]
I am not sure what this `:Security/DefendWorkflows` means. It triggers notification to the es-security team. The change looks fine to me. But I think someone from the core-infra team should take a look at it given this is a change to system indices and [previous PR](https://github.com/elastic/elasticsearch/pull/92640) seems to suggest so. I tagged the PR with the `:Core/Infra/Core` label.

**Comment by aleksmaus** [comment]
> I am not sure what this `:Security/DefendWorkflows` means. It triggers notification to the es-security team. The change looks fine to me. But I think someone from the core-infra team should take a look at it given this is a change to system indices and [previous PR](https://github.com/elastic/elasticsearch/pull/92640) seems to suggest so. I tagged the PR with the `:Core/Infra/Core` label.

Yeah I was not sure what label I should have used for the team for this ticket and without the label PR checks were failing.
Feel free to correct however it needs to be done.

**Pull request merged**
